main/pipeline/pipeline.cc

```diff
@@ -1264,4 +1264,43 @@ vector<ast::ParsedFile> autogenWriteCacheFile(const core::GlobalState &gs, const
 #endif
 }
 
+void printUntypedBlames(const core::GlobalState &gs, const UnorderedMap<long, long> untypedBlames,
+                        const std::string &blameFilePath) {
+#ifndef SORBET_REALMAIN_MIN
+
+    rapidjson::StringBuffer result;
+    rapidjson::Writer<rapidjson::StringBuffer> writer(result);
+
+    writer.StartArray();
+
+    for (auto e : untypedBlames) {
+        auto sym = core::SymbolRef::fromRaw(e.first);
+        if (!sym.exists() || !sym.loc(gs).exists()) {
+            continue;
+        }
```

Comment:
What do you think about including these? I think that knowing how many blind spots we have is value. e.g., the `::<none>` results in your earlier reports give us an approximate estimate of how much untyped can't be blamed due to things like `untypedUntracked` and `T.unsafe` etc.

Reply:
And if we do decide not to include this, in that case let's opt to filter them out before they get into the mapping (i.e., in errorClassForUntyped), so that we're not doing extra work.

Reply:
Done, these are included now; the `path` will just be `<none>` for these

I'm assuming `.owner` will always exist for a `SymbolRef` though

Reply:
Yeah I'm pretty sure that `::<root>` has itself as its owner, so everything is owned by `::<root>` transitively (nothing is unowned)